modules/SCsub

```diff
@@ -2,23 +2,30 @@
 
 Import('env')
 
-env_modules = env.Clone()
+static_build = env["platform"] == "iphone" #or env['static']
```

Comment:
I'd prefer that moved after the `Export('env_modules')` for consistency with other `SCsub`, where import, clone and export are always the first operations.

Reply:
The `env['static']` part was used for testing I guess?

BTW I wonder if `static` is a good name for that. I'm not sure what would be a good name, but we do link everything statically in the `not static_build` case. Maybe `modular_libs` or similar?